scripts/manpage_generation.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Directory to store the generated manpages
+MANPAGE_DIR="man"
+
+VER=$(grep -oP '^Version:\s+\K\S+' packaging/convert2rhel.spec)
+
+# Generate a file with convert2rhel synopsis for argparse-manpage
+python -c 'from convert2rhel import toolopts; print("[synopsis]\n."+toolopts.CLI.usage())' > man/synopsis
+
+# Generate the manpage using argparse-manpage
+PYTHONPATH=. argparse-manpage --pyfile man/__init__.py --function get_parser --manual-title="General Commands Manual" --description="Automates the conversion of Red Hat Enterprise Linux derivative distributions to Red Hat Enterprise Linux." --project-name "convert2rhel $Ver" --prog="convert2rhel" --include man/distribution --include man/synopsis > "$MANPAGE_DIR/convert2rhel.8"
```

Comment:
Issue is still here, we need $VER not $Ver, otherwise it isn't the same as on line 6

```suggestion
PYTHONPATH=. argparse-manpage --pyfile man/__init__.py --function get_parser --manual-title="General Commands Manual" --description="Automates the conversion of Red Hat Enterprise Linux derivative distributions to Red Hat Enterprise Linux." --project-name "convert2rhel $VER" --prog="convert2rhel" --include man/distribution --include man/synopsis > "$MANPAGE_DIR/convert2rhel.8"
```

Reply:
whoops my apologies will fix now!

Reply:
@SpyTec fixed and pushed